app/assets/stylesheets/open_project_global/_variables.sass

```diff
@@ -189,3 +194,5 @@ $user-avatar-mini-border-radius:                50px !default
 $user-avatar-mini-width:                        20px !default
 
 $select-element-padding:                        3px, 24px, 3px, 3px
+
+$repository-breadcrumbs--color:                 #78818A
```

Comment:
First: not sure if this belongs into the global variables, but be it...

What I do not understand (but sadly this is what the visual says) is why we need to introduce a new color here. Now we have three colors:

![three-colors](https://cloud.githubusercontent.com/assets/453584/9658898/2a06d9f6-524d-11e5-8d18-8355a7e97e08.PNG)

* root
* clickable/linked directory (ancestor)
* current directory/file (no link)

when we would only need two colors:

![two-colors](https://cloud.githubusercontent.com/assets/453584/9658944/6e3824fe-524d-11e5-804f-0c641d07f8e5.PNG)

* clickable
* non-clickable

The new color does exist nowhere else (*inconsistency*) and I am asking myself if it is the same color on magenta themed OpenProject instances.

Bonus: The underline color is currently wrong:

![wrong underline](https://cloud.githubusercontent.com/assets/453584/9659007/d9762176-524d-11e5-8590-22b4ad4ab101.png)